lib/logthrdest/logthrdestdrv.c

```diff
@@ -743,11 +743,29 @@ log_threaded_dest_worker_start(LogThreadedDestWorker *self)
   return main_loop_threaded_worker_start(&self->thread);
 }
 
+static void
+_init_queue_sck_builder(LogThreadedDestWorker *self, StatsClusterKeyBuilder *builder)
+{
+  stats_cluster_key_builder_add_label(builder, stats_cluster_label("id", self->owner->super.super.id ? : ""));
+  stats_cluster_key_builder_add_label(builder, stats_cluster_label("driver_instance",
+                                      self->owner->format_stats_instance(self->owner)));
+
+  gchar *worker_index_str = g_strdup_printf("%d", self->worker_index);
```

Comment:
Optional: we could just use a buffer on stack (or scratch_buffers), instead of heap-allocating this small number.

Reply:
Thanks, fixed!